recipes/svaba/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+rm -fr SeqLib
+git clone --recursive https://github.com/walaj/SeqLib.git
```

Comment:
why do you need this special seqlib version?

Reply:
When downloading the [release](https://github.com/walaj/svaba/archive/1.1.0.tar.gz), SeqLib is not included. There is just an empty folder. I just wanted to make sure that the version matched the one at the time the release was created. Maybe that is not necessary.